index.html

```diff
@@ -5,8 +5,149 @@
+          <li class="adadvantages-item">
+            <h4 class="advantages-title">Там есть мост дьявола</h4>
+            <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+        </ul>
+      </section>
+      <section class="invitation">
+        <h2 class="visually-hidden">Приглашение в Седону</h2>
+        <h3 class="invitation">Приезжайте в Седону отдохнуть в комфорте и уюте!</h3>
+        <p class="invitation">Опытный персонал и качественное обслуживание!</p>
+      </section>
+      <section class="recommendations-list">
+        <h2 class="recommendations-list-header visually-hidden">Рекомендации для туристов</h2>
+          <ul class="recommendations-list">
+            <li class="recommendations-item">
+              <h4 class="recommendations-title">Жильё</h4>
+              <p class="recommendations-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+            </li>
+            <li class="recommendations-item">
+              <h4 class="recommendations-title">Еда</h4>
+              <p class="recommendations-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+            </li>
+            <li class="recommendations-item">
+              <h4 class="recommendations-title">Сувениры</h4>
+              <p class="recommendations-description">Не только китайского, но и настоящего местного производства!</p>
+            </li>
+          </ul>
+      </section>
+      <section class="booking">
+        <h2 class="booking-header visually-hidden">Бронирование гостиницы в Седоне</h2>
+        <h3 class="booking">Заинтересовались?</h3>
+        <p class="booking">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <form class="booking-button" method="post">
+              <input class="booking-button" type="submit" name="booking" value="Поиск гостиницы в Седоне">
+          </form>
+      </section>
+      <section class="mailing">
+         <h2 class="mailing-header">Подпишитесь на рассылку</h2>
+         <p class="mailing-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
+         <form class="mailing-button" method="get">
+              <input class="mailing-button" type="text" name="mail" value="Ваш e-mail">
+              <input class="mailing-button" type="submit" name="subscription" value="Подписаться">
+         </form>
+      </section>
+    </main>
+    <footer class="page-footer">
+      <section class="footer-social">
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <ul class="footer-list">
+          <li class="footer-item">
+            <a class="footer-link" href="https://vk.com/htmlacademy">
+            <img class="footer-image" src="#" alt="Иконка Вконтакте">
+            </a>
+          </li>
+          <li class="footer-item">
+            <a class="footer-link" href="https://t.me/htmlacademy ">
```

Comment:
Пробельчик можно и убрать лишний )
```suggestion
            <a class="footer-link" href="https://t.me/htmlacademy">
```